xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -821,16 +826,13 @@ void CGUIWindowVideoBase::GetContextButtons(int itemNumber, CContextButtons &but
   CGUIMediaWindow::GetContextButtons(itemNumber, buttons);
 }
 
-bool CGUIWindowVideoBase::OnPlayStackPart(int itemIndex, unsigned int partNumber)
+bool CGUIWindowVideoBase::OnPlayStackPart(const std::shared_ptr<CFileItem>& item,
```

Comment:
When I built an early dialog, it looked like this, with `const std::shared_ptr<CFileItem>& item` copied from an even earlier dialog and passed around everywhere. I refactored it to pass around `const CFileItem& item` and the result was more concise code without an extra shared_ptr dependency everywhere.